acta-orthopaedica.csl

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+  <info>
+    <title>Acta Orthopaedica</title>
+    <id>http://www.zotero.org/styles/acta-orthopaedica</id>
+    <link href="http://www.zotero.org/styles/acta-orthopaedica" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://www.zotero.org/styles/cell" rel="template"/>
+    <link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
+    <author>
+      <name>Charles Parnot</name>
+      <email>charles@mekentosj.com</email>
+      <uri>http://twitter.com/cparnot</uri>
+    </author>
+    <author>
+      <name>Max Gordon</name>
+      <email>max@gforge.se</email>
+      <uri>http://gforge.se</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
+    <eissn>1745-3682</eissn>
+    <issnl>1745-3674</issnl>
+    <summary>Vancouver style adapted to Acta Orthopaedica</summary>
+    <updated>2017-07-29T05:53:53+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <date form="text" delimiter=" ">
+      <date-part name="year"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="day"/>
+    </date>
+    <terms>
+      <term name="retrieved">available</term>
+      <term name="section" form="short">sect.</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="19" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
delete the et-al-min here. If the setting in the bibliography (70/60) isn't right, move it to the bibliography.